[2024-10-01T12:20:02.648+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-01T12:20:02.663+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: transform_data.success manual__2024-10-01T12:19:59.015870+00:00 [queued]>
[2024-10-01T12:20:02.675+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: transform_data.success manual__2024-10-01T12:19:59.015870+00:00 [queued]>
[2024-10-01T12:20:02.676+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-10-01T12:20:02.691+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): success> on 2024-10-01 12:19:59.015870+00:00
[2024-10-01T12:20:02.697+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=238) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-01T12:20:02.699+0000] {standard_task_runner.py:72} INFO - Started process 240 to run task
[2024-10-01T12:20:02.699+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'transform_data', 'success', 'manual__2024-10-01T12:19:59.015870+00:00', '--job-id', '182', '--raw', '--subdir', 'DAGS_FOLDER/transform_data.py', '--cfg-path', '/tmp/tmp4orlejnz']
[2024-10-01T12:20:02.701+0000] {standard_task_runner.py:105} INFO - Job 182: Subtask success
[2024-10-01T12:20:02.751+0000] {task_command.py:467} INFO - Running <TaskInstance: transform_data.success manual__2024-10-01T12:19:59.015870+00:00 [running]> on host 9ff0d37a02a4
[2024-10-01T12:20:02.838+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='transform_data' AIRFLOW_CTX_TASK_ID='success' AIRFLOW_CTX_EXECUTION_DATE='2024-10-01T12:19:59.015870+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-01T12:19:59.015870+00:00'
[2024-10-01T12:20:02.840+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-01T12:20:02.842+0000] {logging_mixin.py:190} INFO - This DAG is running well
[2024-10-01T12:20:02.843+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-01T12:20:02.853+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-01T12:20:02.855+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=transform_data, task_id=success, run_id=manual__2024-10-01T12:19:59.015870+00:00, execution_date=20241001T121959, start_date=20241001T122002, end_date=20241001T122002
[2024-10-01T12:20:02.916+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-01T12:20:02.930+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-01T12:20:02.937+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
